corehq/apps/builds/README.rst

```diff
@@ -82,7 +82,7 @@ Try accessing ``http://localhost/a/domain`` and see if it works. nginx should
 
 proxy all requests to localhost to your django server. You should also be able
 
-to access ``http://your_ip_address/a/domain`` from a phone or other device on the
+to access ``http://your_ip_address:8000/a/domain`` from a phone or other device on the
```

Comment:
you don't need this port if you are using nginx

Reply:
Hey Simon,
When I was setting this up, this did not work without the port.
I think it's because nginx conf by default has setup for localhost, and since we are running runserver with `./manage.py runserver 192.168.1.5:8000`, nginx does not come into play.
It should work without port if localhost is changed with the IP address in the nginx conf, as mentioned later. So I modified this based on if someone is following this documentation in the same order as written.

Feel free to do a quick check locally to confirm or suggest alternate.

Reply:
The section of the doc you are updating is talking specifically about using nginx.

Reply:
Just rechecked this. Its correct that we should not need port. Updated this later to move it after nginx reload though.